[2023-06-01T11:22:15.580+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-06-01T11:22:15.753+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2023-06-01T11:22:15.756+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:22:15.764+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:22:15.772+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:22:15.891+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-19 00:00:00+00:00
[2023-06-01T11:22:15.929+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-04-19T00:00:00+00:00', '--job-id', '419', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpyfmy3c7a']
[2023-06-01T11:22:15.917+0000] {standard_task_runner.py:55} INFO - Started process 1408 to run task
[2023-06-01T11:22:15.936+0000] {standard_task_runner.py:83} INFO - Job 419: Subtask insert_into_table
[2023-06-01T11:22:16.402+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-19T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:22:16.647+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-04-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-19T00:00:00+00:00
[2023-06-01T11:22:16.649+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:22:16.675+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:22:18.087+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:22:18.135+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-19', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:22:18.143+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:22:18.178+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220419T000000, start_date=20230601T112215, end_date=20230601T112218
[2023-06-01T11:22:18.294+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:22:18.377+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
